script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -87,15 +87,15 @@ It is an Autumn morning in October, a time when all traces of summer have disapp
 <Page24>
 Thinking about it, I've been gazing at this scenery for almost a year and a half.
 This is the last time I will see this scenery on my train ride to school.
-I think back on the past seven years, tracing the landscape as I pass by.
+I rethink of the past seven years, tracing the landscape as I pass by.
 <Page25>
 When I was ten years old, I recovered from a serious accident that should have killed me instantly.
-I met Sensei.
+I have met Sensei.
 Then, I ended up living at the Arima household.
 And so the Autumn of my second year of high school has arrived.
 <Page26>
-Nothing special happened since then like Sensei said, and as long as I wear the glasses she gave me, I don't see any lines.
-I believed that I had spent my days living a precious, ordinary life.
+The special event that Sensei mentioned at the time of parting did not happen at all, and as long as I wear the glasses she has given me, I won't see any line at all.
```

Comment:
Hrm, I'm assuming the jp text here made specific use of the word 'event'? The prev version is a bit more 'natural' here, though it does reinterpret the precise vocab. If you want to keep event, maybe "The special event that Sensei told ['warned' would feel better here, but I assume isn't in src text] me of when we parted did not come to pass"?

Reply:
Also I think 'line' should stay plural here - you wouldn't say "I haven't seen any horse", it would be "I haven't see any horses"